packages/main/src/MultiComboBox.ts

```diff
@@ -581,26 +595,26 @@ class MultiComboBox extends UI5Element {
 
 		if (!isPhone()) {
 			if (filteredItems.length === 0) {
-				this.allItemsPopover.close();
+				this.allItemsPopover?.close();
 			} else {
-				this.allItemsPopover.showAt(this);
+				this.allItemsPopover?.showAt(this);
 			}
 		}
 
 		this.fireEvent("input");
 	}
 
-	_tokenDelete(event) {
-		const token = event.detail.ref;
-		const deletingItem = this.items.find(item => item._id === token.getAttribute("data-ui5-id"));
+	_tokenDelete(e: CustomEvent) {
+		const token: Token = e.detail.ref;
+		const deletingItem = this.items.find(item => item._id === token.getAttribute("data-ui5-id"))!;
 
-		deletingItem.selected = false;
+		(deletingItem as MultiComboBoxItem).selected = false;
```

Comment:
add selected as option property to the interface then this casting will be redundant. also adding a check if this is a real MultiComboBoxItem and then executing the following logic may be a good idea.